executor/include/execution/trait/can_require.hpp

```diff
@@ -0,0 +1,32 @@
+//
+// Created by Shrijit Singh on 14/06/20.
+//
+
+#pragma once
+
+#include <type_traits>
+
+namespace execution {
+
+template <typename Executor, typename Property,
+          typename std::enable_if_t<
+              Property::template is_applicable_property_v<Executor> &&
```

Comment:
Bikeshedding: `is_property_applicable` instead?

Reply:
`is_applicable_property` is what the proposal has used, so using the same naming.